README.md

```diff
@@ -41,10 +41,28 @@ in Internet Explorer on Windows 7 for instance (because of the wrong MIME type).
 
 JavaScript Cookie supports [npm](https://www.npmjs.com/package/js-cookie) and [Bower](http://bower.io/search/?q=js-cookie) under the name `js-cookie`.
 
+- #### Bower
+  ```
+  bower install js-cookie --save
```

Comment:
Yes, I think it's still worth it. While I agree with the ideology of the article, the whole purpose of this PR was to satisfy #333, so being explicit about the the available commands was apart of the fundamental focus.